lib/apt_news.py

```diff
@@ -8,6 +8,8 @@
 from uaclient.config import UAConfig
 from uaclient.daemon import setup_logging
 
+root_logger = logging.getLogger("uaclient")
```

Comment:
"root logger" typically refers to `logging.root`, so let's not call our top-level logger the "root" logger